pastel/reg_ticket.go

```diff
@@ -37,9 +37,9 @@ type RegTicketData struct {
 // ArtTicket is Pastel Art Ticket
 type ArtTicket struct {
 	Version       int       `json:"version"`
-	Author        []byte    `json:"author"`
+	Author        string    `json:"author"`
```

Comment:
why this change is very important to you?

Reply:
please check 
```
"version": integer          // 1
  "author": bytes,            // PastelID of the author (creator)
  "blocknum": integer,        // block number when the ticket was created - this is to map the ticket to the MNs that should process it
  "block_hash": bytes         // hash of the top block when the ticket was created - this is to map the ticket to the MNs that should process it
  "copies": integer,  
```

Reply:
@nvnguyen202 This is necessary figured out after testing.

Reply:
I've tested the flow with cNode. changing it to []byte will break the unmarshal. Please see the screenshot below
![image](https://user-images.githubusercontent.com/46045452/129171400-1f6c2950-553b-4e1a-91a8-6332e9715ce7.png)
![image](https://user-images.githubusercontent.com/46045452/129171444-3024ad88-b738-464c-8b56-0518bf9a17d2.png)

Reply:
if so, let keep it. then try to test it.
but need to change:
```
// Decode hash string to byte
task.artistBlockHash, err = hex.DecodeString(blockInfo.Hash)
```
to :
```
1. change artistBlockHash -> to string type
2. task.artistBlockHash = blockInfo.Hash
```

Reply:
@nvnguyen202 I can happily change this one line but i spent half an hour on it and i think a different PR should do it after looking at things thoroughly because there is already a confusion around this... I came across this FIXME comment in the same file about same thing (block hash format)
probably someone who worked on artworkregister
or I could take a look as well but in the scope of a different PR. One reason is that the mock networks is sending hash which fails to hex decode. (will provide more info on this later). Updated mock networks will introduce other new updates which will break more things that we know about and plan to fix after merging current open PRs. 

![image](https://user-images.githubusercontent.com/46045452/129179104-6862fac0-016c-4bdd-9d93-e9f79bfbb44a.png)

Reply:
@pjenys0 please read my comment above ^^ 
This is why we need a different PR to address this. 
changing this one line is not the solution here. we need to look into it thoroughly.